src/v/datalake/table_definition.h

```diff
@@ -20,6 +20,7 @@ namespace datalake {
 // TODO: rename to redpanda_fields_struct_type?
 iceberg::struct_type schemaless_struct_type();
 iceberg::schema default_schema();
+static constexpr std::string_view rp_struct_name = "redpanda";
```

Comment:
Should be `inline`

Reply:
Oh? Doesn't constexpr imply inline?

Reply:
Not in this context AFAIK: https://en.cppreference.com/w/cpp/language/constexpr

I remember some weird behavior for topic partitions that were static instead of inline, but I now I think those were not constexpr just const... It's probably fine for this case

Reply:
Yeah that was const not constexpr. I really should have a better commit message as to what happened here: https://github.com/redpanda-data/redpanda/commit/b6bf96b9ba16d7bcbdb59c23a9df3a1dcadcdc

I do remember for this example one translation unit had an empty string version of model::tp but anyways it's probably not a problem with string view and constexpr

Reply:
Wild stuff.... Updated anyway!